javascript/javascript1/week1/housy-pricey.js

```diff
@@ -0,0 +1,23 @@
+let volumeInMeters=8*10*10;
+let gardenSizeInM2=100;
+let peterHouseCost=2500000;
+
+let housePrice = volumeInMeters * 2.5 * 1000 + gardenSizeInM2 * 300;
```

Comment:
This is a good example of using `let` not `const` because you assigned new values after.
Good job